app/main.py

```diff
@@ -14,4 +14,5 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    phrase = phrase.lower()
+    return phrase.count(letter.lower())
```

Comment:
You can make it in 1 line only with a return statement. Please, fix it for better code-style

Reply:
Already fixed .